images/nginx-slim/README.md

```diff
@@ -1,3 +1,3 @@
 The `nginx-slim` image has moved to the
```

Comment:
That looks like the `nginx` image. We should probably remove the nginx-slim directory entirely?

Reply:
@foxish Thanks for your review, that makes sense.